docs/Admin.md

```diff
@@ -507,4 +507,59 @@ function AsyncResources() {
 }
 ```
 
+### Dynamically create, update, and delete `<Resource>`
```

Comment:
Now that I think about it, I don't think this addition is relevant. We already explain how to add Resources dynamically based on an HTTP request earlier in this chapter. Your addition only explains another way to do it, which is only useful if people want to recreate ra-no-code. And we don't want to explain too many ways to do things, to encourage reimplementing existing modules, or to solve every possible userland requirement in the documentation.

Reply:
Ok. I can remove this part for now.

Are we going to have the `ra-no-code` documentation mentioned somewhere else?

Reply:
When we feel that it's ready, yes. For now, it's not yet ready.